themes/mongodb/src/css/suggestion.css

```diff
@@ -0,0 +1,81 @@
+html {
```

Comment:
I feel like this rule doesn't belong in this file; our CSS is kind of a hot mess, but `basic.css` might be a better place